README.md

```diff
@@ -25,25 +29,31 @@ Require it:
 const { Logger } = require('@hmcts/nodejs-logging')
 ```
 
-Set the logging configuration. Should be done once at application startup:
-
-```javascript
-Logger.config({ 
-    microservice: 'your-service-name', 
-    team: 'your-team',
-    environment: 'some-environemnt'
-})
-```
-
 Then you can create a logger instance and use it to log information:
 
 ```javascript
 const logger = Logger.getLogger('app.js') // app.js is just an example, can be anything that's meaningful to you
+```
+usage are:
 
+```
 logger.info({
   message: 'Yay, logging!'
 })
 ```
+or
+```
+logger.log({
+  level: 'info',
+  message: 'What time is the testing at?'
+});
+
+ Outputs:
+{ level: 'info',
+  message: 'What time is the testing at?',
+  label: 'right meow!',
```

Comment:
Where did label come from? Shouldn't that be `app.js` as you instantiated above?